### ISSUE TITLE
Add git show capability

### ISSUE THREAD

**Issue by shumashv1** [ZoomerAnalytics/git-xltrail#29]
Git diff only works in a branch prior to a commit.  Could the diff be accessed in some way to show the changes after the commit using git show?

**Comment by fzumstein** [comment]
You're right, git-xltrail currently doesn't play with `diff show`. However, using something like `git diff HEAD~1` should work as a workaround for now?

**Comment by shumashv1** [comment]
That works.  How about adding something simple like this?

#show.py
Import os
os.system(‘git diff HEAD~1')

Ugly, I know.
ShumashGeely
